app/src/main/java/net/squanchy/notification/NotificationCreator.java

```diff
@@ -173,6 +174,7 @@ private String getDisplayedSpeakers(Event event) {
     }
 
     private String createSummaryTitle(int talksCount) {
-        return context.getString(R.string.event_notification_count_starting, talksCount);
+        String template = context.getString(R.string.event_notification_count_starting);
```

Comment:
Maybe? Lint was flagging it tho. TBH it makes literally no difference in this case, there's only that many ways a small number can be formatted :D